### PR TITLE
ASMFT 222-225

### PR DESCRIPTION
**Pull request by roppel5ls**
This commit creates the basis for a checkout scripts. I have 4 files of actual "scripts" that contain an automated user going through the purchase path of LS, IDS, WALS, and Primerica for a CC/BD transaction. 

There is a file I built to house my modules where you will find the actual scripts I have written as functions.

@TrentonWalkup please review the tools file as I made some changes.

Ask me if you have any questions.. It's a big one

### CODE REVIEW

File: demo/checkout_mods.js [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+  enterText('//input[@name="askPassword"]', 'Testing01')
+  enterText('//input[@name="ask_confirm_password"]', 'Testing01')
+  scroll('#askData__next > button:nth-child(1)')
+}
+
+//End Associate Sections
+
+
+//5.) Payments
+var bankName = 'VISION BANK NA'
```
[the remark this answers is not in the document]
Comment (roppel5ls) [reply demo/checkout_mods.js]: Yes we are asserting that the transit number I put in actually returns the correct bank name from the API

File: demo/checkout_mods.js [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+var walsIDTPlan = 'IDShield Family'
+var walsIDTPrice = '$24.95'
+var primericaLGLPlan = 'Primerica Legal Plan with Emergency Access'
+var primericaLGLPrice = '$26.00'
+var primericaIdsPlan = 'ID Theft Defense Family Plan'
+var primericaIdsPrice = '$22.00'
+
+var corpLsSelectRegionPlanCC = function () {
+  click('#select_4')
+  scroll('md-option[value="OK"')
```
Comment (TrentonWalkup) [inline demo/checkout_mods.js]: did we decide not to use a random state? I can't remember?

File: demo/tools.js [hunk 3]
```diff
@@ -25,6 +25,16 @@ global.assertTitle = function(title){
   assert.equal(a, title)
 }
 
+global.assertUrl = function(url){
+  var a = browser.getUrl()
+  assert.equal(a, url)
+}
+
+global.assertValue = function(selector, value){
```
Comment (TrentonWalkup) [inline demo/tools.js]: good

File: demo/checkout_mods.js (hunk 2 above)
Comment (roppel5ls) [reply demo/checkout_mods.js]: Yes, I used 'OK' so I did not have to account for differences between licensed states, or states that may not have all the plans/supplements available.

I want to go back in the future and make it select a random state/random plan.. But MVP for site functionality we discussed having a set State/Plan combo.

**Comment by robertvazquezlegalshield** [comment]
LGTM

**Comment by roppel5ls** [comment]
Approved by @TrentonWalkup and @robertvazquezlegalshield. Merging to Master

**Pull request merged**